### PR TITLE
Add release notes for v0.5.0

### PR DESCRIPTION
**Pull request by tpantelis**
Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

### CODE REVIEW

File: src/content/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,26 @@ pre = "<b>5. </b>"
...
+
+* **Lighthouse** has been modified per the [Kubernetes Multicluster Services KEP][MCS KEP] as follows:
+  - The `MultiClusterService` resource has been replaced by `ServiceImport`.
+  - The `ServiceExport` resource is now updated with status information as lifecycle events occur.
+* **Lighthouse** now allows a `ServiceExport` resource to be created prior to the associated `Service`.
+* Network discovery was moved from `subctl` to the subamriner operator.
+* Several new commands were added to `subctl`: `export service`, `show versions`, `show connections`, `show networks`,
+  and `show gateways`.
+* The **Globanet** configuration has been moved from the `broker-info.subm` file to a `ConfigMap` resource stored on the
+  broker cluster.
```
Comment (sridhargaddam) [inline src/content/releases/_index.en.md:21]: Can you please include the following as well.

subctl now supports joining multiple clusters in parallel without having to explicitly specify the globalnet-cidr for the cluster. Globalnet-cidr will be automatically allocated by subctl for each cluster.

File: src/content/releases/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,26 @@ pre = "<b>5. </b>"
...
+* The separate `--operator-image` parameter has been removed from `subctl join` and the `--repoository` and `--version`
+  parameters are now used for all images.
+* The submariner operator status now includes `Gateway` information.
+* Closed technical requirements for submariner to become a CNFC project, including _Developer Certificate of Origin_ compliance,
```
Comment (nyechiel) [inline src/content/releases/_index.en.md:25]:
```suggestion
* Closed technical requirements for Submariner to become a CNFC project, including _Developer Certificate of Origin_ compliance,
```

File: src/content/releases/_index.en.md [hunk 3]
```diff
@@ -5,6 +5,26 @@ pre = "<b>5. </b>"
...
+
+> This release mainly focused on continuing the alignment of Lighthouse's service discovery support with the [Kubernetes Multicluster
+> Services KEP][MCS KEP].
+
+* **Lighthouse** has been modified per the [Kubernetes Multicluster Services KEP][MCS KEP] as follows:
+  - The `MultiClusterService` resource has been replaced by `ServiceImport`.
+  - The `ServiceExport` resource is now updated with status information as lifecycle events occur.
+* **Lighthouse** now allows a `ServiceExport` resource to be created prior to the associated `Service`.
+* Network discovery was moved from `subctl` to the subamriner operator.
+* Several new commands were added to `subctl`: `export service`, `show versions`, `show connections`, `show networks`,
```
Comment (maayanf24) [inline src/content/releases/_index.en.md:18]: Might worth mention that 'show networks' is replacing the old command 'subctl info'.
Another new command is 'show endpoints'

File: src/content/releases/_index.en.md [hunk 4]
```diff
@@ -5,6 +5,29 @@ pre = "<b>5. </b>"
...
+* **Lighthouse** has been modified per the [Kubernetes Multicluster Services KEP][MCS KEP] as follows:
+  - The `MultiClusterService` resource has been replaced by `ServiceImport`.
+  - The `ServiceExport` resource is now updated with status information as lifecycle events occur.
+* **Lighthouse** now allows a `ServiceExport` resource to be created prior to the associated `Service`.
+* Network discovery was moved from `subctl` to the Submariner operator.
+* Several new commands were added to `subctl`: `export service`, `show versions`, `show connections`, `show networks`,
+  `show endpoints`, and `show gateways`.
+* The `subctl info` command has been removed in lieu of the new `show networks` command.
+* The **Globanet** configuration has been moved from the `broker-info.subm` file to a `ConfigMap` resource stored on the
+  broker cluster.
```
[the remark this answers is not in the document]
Comment (tpantelis) [reply src/content/releases/_index.en.md]: Done

**Pull request merged**